subsys/net/lib/http/http_server.c

```diff
@@ -311,7 +313,7 @@ static inline void new_client(struct http_ctx *ctx,
 	struct net_context *net_ctx;
 	const char *type_str = "HTTP";
 
-	net_ctx = get_server_ctx(app_ctx, ctx->addr);
+	net_ctx = get_server_ctx(app_ctx, NULL);
```

Comment:
This will always return NULL now.  See get_server_ctx() in http_server.c

Reply:
True, I will fix this.